infinispan-client-quickstart/src/test/java/org/acme/infinispanclient/InfinispanGreetingResourceTest.java

```diff
@@ -0,0 +1,37 @@
+package org.acme.infinispanclient;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.FixedHostPortGenericContainer;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import static org.hamcrest.Matchers.is;
+
+@Testcontainers
+@QuarkusTest
+class InfinispanGreetingResourceTest {
+
+    @Container
+    public static final GenericContainer INFINISPAN = new FixedHostPortGenericContainer(
+            "jboss/infinispan-server:10.0.0.Beta3")
```

Comment:
Should it use the Final now? We have moved to the Final.

Reply:
Need to check, before it was not working with final nor with any other version than this one.

Reply:
@gsmet Does not work with 1.0.0-Final, 1.0.0-Final-1, 1.0.1-Final-1